scripts/ansible/roles/windows/az-dcap-client/tasks/validation.yml

```diff
@@ -19,36 +19,37 @@
   register: file
   failed_when: file.stat.isreg == False
 
-- name: Azure DCAP Client | Check for the LC driver registry key
```

Comment:
Was this file intentionally modified? It looks to be white space changes without any logic, if so, could we revert.

Reply:
Yes. These validation tasks only need to run in this case:
```
when: launch_configuration is defined and launch_configuration == 'SGX1FLC'
```

The Ansible `launch_configuration` parameter was added in this pull request for a clear separation between Windows machines configured:
* Without SGX driver installed: `SGX1-NoDriver` or `SGX1FLC-NoDriver`. Windows machines could either have driver installed via Windows Update or they could simply lack SGX support.
* With SGX support (the driver is installed): `SGX1`.
* With SGX1FLC support (the driver is installed): `SGX1FLC`.

Reply:
Oh ok, that makes sense. Thanks for explaining